pytorch_lightning/profiler/xla.py

```diff
@@ -65,6 +66,10 @@ class XLAProfiler(BaseProfiler):
     def __init__(self, port: int = 9012) -> None:
         """This Profiler will help you debug and optimize training workload performance for your models using Cloud
         TPU performance tools."""
+        if not _TPU_AVAILABLE:
+            raise MisconfigurationException("`XLAProfiler` is only supported on TPUs")
+        if not _TORCH_GREATER_EQUAL_1_8:
+            raise MisconfigurationException("`XLAProfiler` is only supported with `torch-xla>=1.8`")
```

Comment:
```suggestion
            raise MisconfigurationException("`XLAProfiler` is only supported with `torch-xla >= 1.8`")
```
can we throw a space in there 😅 
(it would require updating tests though)

Reply:
Sure! Will update that